store.go

```diff
@@ -3417,12 +3417,44 @@ func ReloadConfigurationFile(configFile string, storeOptions *StoreOptions) {
 	}
 }
 
+var (
+	prevStoreOptions *StoreOptions
+	prevMod          time.Time
+	prevMutex        sync.Mutex
+	prevConfigFile   string
+)
```

Comment:
It’s a bit but I’d prefer to have just one single global mutable variable as struct.

Reply:
Ok @giuseppe Change to a struct and then we can merge.